src/Organic/AdminSettings.php

```diff
@@ -227,6 +230,7 @@ public function showSettings() {
                 <?php AdminNotice::showNotices(); ?>
             </div>
             <h1>Organic Settings</h1>
+            <h2>Plugin settings last updated: <?php echo esc_attr( $settings_last_updated->format( 'y-m-d h:i:s T' ) ); ?></h2>
```

Comment:
Maybe someone has suggestions for a better display?

Reply:
I don't think this info needs so much attention. How about putting it under `Save settings` button and wrapping in `<small>` tag?
![image](https://github.com/orgnc/wordpress-plugin/assets/2822102/fd9ff9d5-e7ce-49a2-b443-bebbad4d2fb7)